README.md

```diff
@@ -337,6 +344,90 @@ spec:
 - The operator fetches the secret from GCP Secret Manager and injects it as a
 secret:
 
+```shell
+% make install
+% make deploy
+% kubectl get secret externalsecret-operator-externalsecret-sample -n externalsecret-operator-system \
+  -o jsonpath='{.data.example-externalsecret-key}' | base64 -d
+```
+
+<a name="gitlab-cicd-variables"></a>
+
+## Gitlab CI/CD Variables
+
+<a name="gitlab-cicd-variables-pre"></a>
+
+#### Prerequisites
+- A Gitlab project with a CI/CD variable, who's key is `example-externalsecret-key`
```

Comment:
Hey mate, I got an error while creating a variable with this name: `Variables key can contain only letters, digits and ‘_’.`. 

What about changing the example to `example_externalsecret_key`, with underscores?

![Screenshot 2020-11-12 at 11 47 50](https://user-images.githubusercontent.com/8570364/98931141-f44d2180-24dd-11eb-84e9-c1b7a3b61d6e.png)

Reply:
Oh, I forgot gitlab doesn't support dashes. (I tested with the name test.) Will change